### ISSUE TITLE
“find in files” shortcut doesn't works

### ISSUE THREAD

**Issue by huujuun** [k--kato/vscode-intellij-idea-keybindings#54]
use the shortcut “ctrl+shift+f” , “find in files”  doesn't works

add this in the custom file as belows. It works well.
```
// Place your key bindings in this file to overwrite the defaults
[
        {
        "key": "ctrl+shift+f",
        "command": "workbench.view.search",
        "when": "!searchViewletVisible"
    },

    {
        "key": "ctrl+shift+f",
        "command": "workbench.action.findInFiles",
        "when": "!searchInputBoxFocus"
    },
        {
        "key": "ctrl+shift+f",
        "command": "search.action.focusActiveEditor",
        "when": "searchInputBoxFocus && searchViewletVisible"
    }

]
```

maybe `{ "key": "ctrl+shift+f",          "command": "workbench.view.search" }` is not needed

**Comment by thekalinga** [comment]
@k--kato Any idea why this is happening?

**Comment by k--kato** [comment]
@thekalinga I just fixed and released v0.2.11. Please check it out.
